gitbook-docs/request_response.md

```diff
@@ -0,0 +1,142 @@
+# Request/Response
```

Comment:
Do we need a short intro, raison d'etre for req/resp?

Reply:
I took a shot at it. Seems like it needs more?